mcs/packages/Makefile

```diff
@@ -26,6 +26,9 @@ ROSLYN_FILES_TO_COPY_FOR_MSBUILD = \
 	$(ROSLYN_CSC_DIR)/Microsoft.VisualBasic.Core.targets
 
 ROSLYN_FILES_TO_LINK_FOR_MSBUILD = \
```

Comment:
this is now almost identical with `ROSLYN_FILES_FOR_MONO`. should we remove this one and just link all of `ROSLYN_FILES_FOR_MONO` ?

Reply:
`Microsoft.CodeAnalysis.VisualBasic.dll` is the only one left. Updating to symlink all the files.